Add Progress arc color based on skill to Xp Globes plugin.

Monsterxsync opened the pull request:
-Relocated SkillColor.Java from xptracker plugin to game folder (fits in well here as its where the icons get loaded for orbs.)

A screenshot of this feature in use: 
![screenshot](https://puu.sh/B5Dmt/843945a133.png)

As you can see it uses the already implemented xp tracker colors for consistancy.

Nightfirecat commented:
SkillColor.java shouldn't be in the game directory, it would be much better-suited to the `client/util` directory. (see OSType.java)

Monsterxsync commented:
I'll move it over if you think its better suited there thanks.

Nightfirecat commented on runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java:
```diff
@@ -43,11 +43,22 @@ default boolean enableTooltips()
+	@ConfigItem(
+		keyName = "enableColorBasedSkillOrbs",
+		name = "Progress arc color based on skill",
+		description = "Change the color of the progress arc in the xp orb based on skill being trained.",
```
It might be good to explicitly state that this overrides the "Progress arc color" option. That might help cut down on support questions on this in the future, if added.

Monsterxsync commented:
Ill change up the description and reorder the imports (forgot to do it after moving skillcolor). More descriptions are always good.

Monsterxsync commented:
Added a better description informing users that the feature will override colour selection.
Reordered imports to correct order.
Changed default option to off.

Everything should be good now 👍 .

Nightfirecat commented:
Needs rebase

Nightfirecat commented:
Also, this fixes #2888

Monsterxsync commented:
I'll rebase it when im home next (y)

Monsterxsync commented:
Very strange moved nothing on my end apart from skillcolor must have happened during the rebase I'll have another look to why its moved these.

Monsterxsync commented:
something crazy was going on my fork, all fixed up now. (i hope).
